src/blocks/Biography/Biography.tsx

```diff
@@ -0,0 +1,70 @@
+import { Avatar, AvatarFallback, AvatarImage } from '@/components/ui/avatar'
+import { Card, CardContent } from '@/components/ui/card'
+import { Popover, PopoverContent, PopoverTrigger } from '@/components/ui/popover'
+import { format, parseISO } from 'date-fns'
+import type { BiographyBlock as BiographyBlockProps } from 'src/payload-types'
+
+type Props = BiographyBlockProps
+
+export const BiographyBlock = ({ biography }: Props) => {
+  if (typeof biography !== 'object' || typeof biography.photo !== 'object') {
+    // TODO: figure out how to handle this - do we have access to the Payload client?
```

Comment:
We should have access to the payload client with a custom component. When will biography not be an object?

Reply:
> We should have access to the payload client with a custom component.

I couldn't figure out how to get access to the `payload` client in a custom block, specifically - it looks like we need to pass it from our `page.tsx` into `<RenderBlocks/>` and clean up our demultiplexing in `<RenderBlocks/>`. I can take a stab at that.

> When will biography not be an object?

If the output of `payload.find()` doesn't have `depth` set sufficiently in the top-level query, these references will just be IDs.

Reply:
Please see the latest commit. This is pretty hairy.